main.go

```diff
@@ -288,6 +290,11 @@ func main() {
 		go setupIPMasq(config, bn)
 	}
 
+	// Set up ipForward if needed
+	if opts.ipForward {
+		go setupIPForward()
```

Comment:
why is this a goroutine? Seems like the main thread can end before this goroutine is done since it's not part of the waitgroup that waits at the end of the `main()`. So either we can serialize this or make the main thread wait for it. WDYT? (Same applies to `go setupIPMasq()`)